mobile/android/app/build.gradle

```diff
@@ -49,6 +49,16 @@ android {
         versionName flutterVersionName
     }
 
+    flavorDimensions "default"
+
+    productFlavors {
+        demo {
```

Comment:
nit: its a bit confusion that we have test flavor in ios and a demo flavor on Android. 🙂

Reply:
unfortunately it's a naming issue - in android the flavour cannot start with `test`